ChronoStore/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.17)
+project(chronostore)
+
+find_package(HDF5 REQUIRED)
+
+# Set the build directory to a subdirectory called "build"
+set(CMAKE_BINARY_DIR ${CMAKE_SOURCE_DIR}/build)
+
+include_directories(include)
+include_directories(${HDF5_INCLUDE_DIRS})
```

Comment:
What are the compiler flags? Can you add '-O3' option to CXX flags?

Reply:
Done. Except for flag to turn debug mode